docs/source/search/granules.ipynb

```diff
@@ -0,0 +1,404 @@
+    "help(maap.searchGranule)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see from the result `maap.searchGranule` accepts a limit keyword which limits the number of results from CMR. `maap.searchGranule()` also accepts any additional search parameters that are included in CMR. For a list of accepted parameters, please refer to the [CMR Search Granules API reference](https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html#granule-search-by-parameters)\n",
+    "\n",
+    "It is important to note that _the default limit on results from the MAAP API is 20_. To increase the number of results we will specify a variable and use it in later queries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get at max 500 results from CMR\n",
+    "MAXRESULTS = 500"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we will explore search options including:\n",
+    "\n",
+    "1. Searching by Collection Concept ID\n",
+    "2. Searching by temporal filter\n",
+    "3. Searching by spatial filter\n",
+    "4. Using the results from one search as inputs into another\n",
+    "5. Searching by additional attributes\n",
+    "\n",
+    "To make things consistent we will focus on the [IceSat-2/ATLAS Land and Vegetation Height dataset](https://dx.doi.org/10.5067/ATLAS/ATL08.002) (except for searching by additional attributes)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Searching by Collection Concept ID\n",
+    "\n",
+    "Here we will search by a unique ID that is given to CMR collections. You can find the collection IDs for all of the collections in MAAP in a table within the documentation: https://maap-project.readthedocs.io/search/cmr_collection_table.html\n",
+    "\n",
+    "It is recommended to begin the search with the Collection Concept ID as this is a specific unique identifier for a collection and will avoid ambiguity when searching by a long name or short name."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'Got 500 results'\n"
+     ]
+    }
+   ],
+   "source": [
+    "COLLECTIONID = 'C1200116818-NASA_MAAP' # specify the collection id for the ATLAS dataset\n",
+    "\n",
+    "results = maap.searchGranule(concept_id=COLLECTIONID,limit=MAXRESULTS)\n",
+    "pprint(f'Got {len(results)} results')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We were able to get 500 results! There are most likely more than 500 granules in search results, but remember we limited the results to 500 granules. The results from the MAAP API is a list of granules where each element in the list is the metadata for that particular granule.\n",
```

Comment:
Ok to use result rather than output.  But it should be "The result...is a list"

Reply:
Just pushed updated. Please verify and resolve